plugin/go.vim

```diff
@@ -21,6 +21,7 @@ let s:packages = [
       \ "github.com/fatih/gomodifytags",
       \ "github.com/zmb3/gogetdoc",
       \ "github.com/josharian/impl",
+      \ "github.com/dominikh/go-tools/tree/master/cmd/keyify",
```

Comment:
This is a wrong import path, should be: `github.com/dominikh/go-tools/cmd/keyify`